core/src/main/scala/scala/pickling/spi/PicklerRegistry.scala

```diff
@@ -56,23 +56,22 @@ trait PicklerRegistry {
     *             In those situations, the pickler should be able to handle arbitrary (existential) type parameters.
     * @param p  The pickler to register.
     */
-  def registerPickler[T](key: String, p: Pickler[T]): Unit
-  def registerPickler[T](p: Pickler[T]): Unit = registerPickler(p.tag.key, p)
+  def registerPickler(key: String, p: Pickler[_]): Unit
+  def registerPickler(p: Pickler[_]): Unit = registerPickler(p.tag.key, p)
```

Comment:
I do not think we should remove the type Tags.   Why have it be existentially qualified when it does not need to be?

I could just be knee-jerking, but I know from sbt code, you wind up with far less compiler errors when you stick with universal quantification when you have no reason to existentially quantify.   Could be many of these erroneous type errors have been fixed, but I still don't see the above as an improvement.